fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/expressions/InsertExpression.java

```diff
@@ -192,12 +189,7 @@ private static Type.Record computeResultType(@Nonnull final Type inType, @Nonnul
     }
 
     @Nonnull
-    private static Value makeComputationValue(@Nonnull final Quantifier inner, @Nonnull final Type targetType) {
```

Comment:
The now not-used method `computeResultType` is not used anymore, this is causing PMD check to fail.

Reply:
... pushed a commit to address that.